SimPEG/DataMisfit.py

```diff
@@ -69,60 +53,118 @@ class l2_DataMisfit(BaseDataMisfit):
 
     .. math::
 
-        \mu_\\text{data} = {1\over 2}\left| \mathbf{W}_d (\mathbf{d}_\\text{pred} - \mathbf{d}_\\text{obs}) \\right|_2^2
+        \mu_\\text{data} = {1\over 2}\left|
+        \mathbf{W}_d (\mathbf{d}_\\text{pred} -
+        \mathbf{d}_\\text{obs}) \\right|_2^2
 
     """
 
+    std = 0.05  #: default standard deviation if not provided by survey
```

Comment:
we should consider removing these defaults. These are important things to consider in an inversion. However that is outside the scope of this pr.